packages/@uppy/server-utils/src/Provider.js

```diff
@@ -52,4 +52,27 @@ module.exports = class Provider extends RequestClient {
         return res
       })
   }
+
+  static initPlugin (plugin, opts, defaultOpts) {
+    plugin.type = 'acquirer'
+    plugin.files = []
+    if (defaultOpts) {
+      plugin.opts = Object.assign({}, defaultOpts, opts)
+    }
+    if (opts.serverPattern) {
+      const pattern = opts.serverPattern
+      // validate serverPattern param
+      if (typeof pattern !== 'string' && !Array.isArray(pattern) && !(pattern instanceof RegExp)) {
+        throw new TypeError('Provider: the option `serverPattern` must be one of string, Array, RegExp')
```

Comment:
could we use `${plugin.id}` at the start of this message instead of `Provider` so it shows if it's from GoogleDrive or Dropbox or XYZ?